Quintet-iOS/View/CalendarView.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  CalendarView.swift
+//  Quintet-iOS
+//
+//  Created by 옥재은 on 2023/11/10.
+//
+
+import SwiftUI
+
+struct CalendarView: View {
+    @Binding var currentDate: Date
+    @StateObject private var viewModel = DateViewModel()
+    @ObservedObject private var coreDataViewModel = CoreDataViewModel()
+    @State private var selectedDate = Date()
+    @State private var isShowPopup = false
+    @State var currentMonth: Int = 0
+    let days: [String] = ["일", "월", "화", "수", "목", "금", "토"]
+    
+    var body: some View {
+        
+        VStack(spacing: 13) {
+            
+            HStack(spacing: -7){
+                
+                Button(action: {
+                    isShowPopup = true
+                }) {
+                    viewModel.yearMonthButtonTextRecordVer
+                        .padding(.horizontal)
+                        .foregroundColor(Color.black)
```

Comment:
```suggestion
                        .foregroundColor(.black)
```
위와 같이 Color를 생략해도 괜찮을것 같습니다!